### PR TITLE
Add run_interop.sh jenkins script

### PR DESCRIPTION
**Pull request by jtattermusch**
So that jenkins does not have to know the cmdline for tools/run_tests/run_interop_tests.py.

### CODE REVIEW

File: tools/jenkins/run_interop.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+# THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# This script is invoked by Jenkins and runs interop test suite.
+set -ex
+
+# Enter the gRPC repo root
+cd `dirname $0`/../..
```
Comment (stanley-cheung) [inline tools/jenkins/run_interop.sh]: You may want to consider doing `$(dirname $0)` instead.

http://stackoverflow.com/questions/9405478/command-substitution-backticks-or-dollar-sign-paren-enclosed

Comment (jtattermusch) [inline tools/jenkins/run_interop.sh]: Did that along with some more improvements.

**Comment by stanley-cheung** [comment]
LGTM, Jenkins is all red so I hit rebuild.

File: tools/jenkins/run_interop.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# This script is invoked by Jenkins and runs interop test suite.
+set -ex
+
+# Enter the gRPC repo root
+cd $(dirname $0)/../..
+
+tools/run_tests/run_interop_tests.py --l all --s all --cloud_to_prod --use_docker -t -j 8 $@ || true
```
Comment (stanley-cheung) [inline tools/jenkins/run_interop.sh]: actually, shouldn't this be just `-l` and `-s`?

**Comment by jtattermusch** [comment]
The windows problems reported by Jenkins are known issues. This change doesn't influence any tests, so merging.

**Pull request merged**